spec/unit/rest/client_spec.rb

```diff
@@ -50,4 +50,11 @@
       end
     end
   end
+
+  context 'request_id generation' do
+    let(:client_options) { {key: 'appid.keyuid:keysecret', add_request_ids: true} }
```

Comment:
Unusual spacing, in all other places `{` has a space after, and `}` a space before